proof/refine/AARCH64/orphanage/Orphanage.thy

```diff
@@ -97,7 +97,7 @@ lemma no_orphans_disj:
   apply (rule ext)
   apply (unfold no_orphans_def all_active_tcb_ptrs_def
                 is_active_tcb_ptr_def st_tcb_at_neg' typ_at_tcb')
-  apply (auto intro: pred_tcb_at')
+  apply (auto del: pred_tcb_at' intro: pred_tcb_at')
```

Comment:
It doesn't, it just gets rid of the warning `auto` produces from using `intro: pred_tcb_at'`, because `pred_tcb_at'` is already declared as some kind of intro/dest/elim rule.